docs/components/dialogs.md

```diff
@@ -3,11 +3,10 @@ title: "Dialogs"
 layout: detail
 section: components
 excerpt: "Dialogs are modal windows that require interaction."
-iconId: 
+iconId:
 path: /catalog/dialog/
 -->
 
-
 # Dialogs
 
 [Dialogs](https://material.io/components/dialogs) inform users about a task and can contain critical information, require decisions, or involve multiple tasks.
```

Comment:
The types list and composite image need to go a separate "Types" section. Use https://github.com/mingjane-work/doc-material-components/blob/mingjane-doc-branch/dialogs/Flutter/assets/dialogs_hero.png as the hero image.

Reply:
Done